src/plugins/discover/public/application/view_components/canvas/index.tsx

```diff
@@ -49,11 +49,27 @@ export default function DiscoverCanvas({ setHeaderActionMenu, history }: ViewPro
   if (isCallOutVisible) {
     callOut = (
       <EuiFlexItem grow={false}>
+        <EuiPanel hasBorder={false} hasShadow={false} color="transparent" paddingSize="s">
```

Comment:
Why do we have 2 callouts here?

Reply:
that was a merge conflict - should be fixed

Reply:
oh, somehow managed to not commit - let me try again.

Reply:
ha, OK, yeah, this should be fixed - but doublecheck my resolution here.